src/Service/Resolvers/BaseSqlQueryBuilder.cs

```diff
@@ -361,7 +361,9 @@ protected string Build(List<string> columns)
         /// <summary>
         /// Join predicate strings while ignoring empty or null predicates
         /// </summary>
-        /// <returns>returns "1 = 1" if no valid predicates</returns>
+        /// <param name="predicateStrings">Array of predicate strings coming from filters in query string,
+        /// primary key predicates for update operations or database policies.</param>
```

Comment:
why only "update" operations?

Reply:
Oh yeah, will update the comment as the predicates can be for other operations as well, thanks.